docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,556 @@
+
+    - .NET Framework versions from 4.5 and up, with Windows features
+    - Windows Phone
+    - Windows Phone Silverlight
+    - Universal Windows Platform
+    - DNX Core
+    - Xamarin Platforms
+    - Mono 
+
+    Then [refer to this table](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md#mapping-the-net-platform-standard-to-platforms) and choose the version of the `dotnetXX` moniker you need.
+    
+    For example, if you wanted to have compatibility with .NET Core and .NET Framework 4.6, you would pick `dotnet55`:
+
+    ```javascript
+    {
+        "frameworks":{
+            "dotnet55":{}
+        }
+    }
+    ```
+    
+    If you wanted to be compatible with Windows Phone Silverlight 8, you would pick `dotnet51`:
+    
+    ```javascript
+    {
+        "frameworks":{
+            "dotnet51":{}
+        }
+    }
+    ```
+3. If you care about compatibility with the .NET Framework versions 4.0 or below, or need to use .NET Framework APIs not available in .NET Core, skip to the next section.
+
+## How to target both .NET Core and .NET Framework
+
+**NOTE:** This requires Windows and .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
+
+You may need a library to work on .NET Core *and* .NET Framework 4.0 or older.  You may also need to use specific libraries available in .NET 4.5 which are unavailable in .NET Core (such as `SYSTEM.FIXME.SOON`).  For these scenarios, you will need to specifically target the versions of .NET Framework you support.
+
+This is done by adding relevant Target Framework Moniker, such as `net45` to your `project.json` file.
+
+For example, you may want to support .NET Framework 4.0 and 4.5, but use the most up-to-date libraries for your .NET Core target.  Setting up your `project.json` file like this will do the trick:
+
+```javascript
+{
+    "frameworks":{
+        "net40":{},
+        "net45":{},
+        "dotnet55":{
+            "dependencies":{
+                "System.Runtime":"4.0.0-rc1-*"
+            }
+        }
+    }
+}
+```
+
+And that's it!  You can now build the project and see the produced artifacts.
+
+Using DNX:
+
+```
+$ dnu restore
+$ dnu build
+```
+
+Using .NET CLI:
+
+```
+$ dotnet restore
+$ dotnet compile
+```
+
+You will now notice three new entries in your `/bin` folder:
+
+```
+/bin
+   /Debug
+      /net40
+      /net45
+      /dotnet
+```
+
+**NOTE:** This assumes your code will compile across *both* .NET Core and .NET Framework.  Read the section on cross-compiling with `#if`s on how to compile the same file differently for each target if you are using features which are unavailable in some of your targets.
+
+## How to target a Portable Class Library (PCL)
+
+**NOTE:** This requires Windows and .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
```

Comment:
Should we link to a topic that talks about PCL in case I don't know what this is?